### ISSUE TITLE
ui.browseableMessage in HTML mode should allow links to trigger the default browser

### ISSUE THREAD

**Issue by XLTechie** [nvaccess/nvda#11527]
### Is your feature request related to a problem? Please describe.

Links in HTML documents shown in ui.browseableMessages can not be activated.

### Describe the solution you'd like

When viewing some HTML in a ui.browseableMessage with isHtml=True, I think that at the very least, links should be clickable, and should open in the default browser.
This would require recognizing anchors, and probably being aware of `<base href`.

Naturally, there are cases where this would fail, such as when javascript is used to create the href, and probably in various other cases.  However for the most basic situation, a link should open in the default browser in my opinion.

### Additional context

Test case for Python console:

```
message = """<p>A standard paragraph.</p>
<p>A paragraph with a <a href="https://nvaccess.org/">link</a>.</p>
"""
ui.browseableMessage(message, title="A link to NV Access", isHtml=True)
```

**Comment by CyrilleB79** [comment]
On my side, with the console example you gave, the link can be activated and opens the NVAccess webpage in the default browser (Chrome).

**Comment by XLTechie** [comment]
Now I am very confused.

On the system I first tested: nothing happens, no matter what link (http, https) I put there.
Firefox is the default browser on this system.

After your comment @CyrilleB79 I tested another machine.
On that one, Firefox is also the default browser, and is the default for opening .htm and .html files (I double checked).
However when I tested the example there, the page did indeed open--in Internet Explorer 11!  IE had never been opened on that system (I didn't even know it was there), and indeed the first thing it tried to do was have me setup preferences and download Microsoft Edge, although it did have nvaccess.org open in another tab or window.

So now I don't know what to think about this, although obviously my initial understanding was wrong, so I will close the issue.
